frontend/src/scenes/insights/InsightTabs/InsightActionBar.tsx

```diff
@@ -22,6 +23,7 @@ export function InsightActionBar({ filters, annotations, insight, onReset }: Pro
                 onConfirm={() => {
                     window.scrollTo({ top: 0 })
                     onReset ? onReset() : push(`/insights?insight=${insight}`)
+                    posthog.capture('insights tab reset')
```

Comment:
Here we should be able to just call + import the eventUsagelogic action via the useActions hook